src/client.rs

```diff
@@ -274,6 +276,31 @@ impl<B: IntoBuf> ::futures::Stream for Body<B> {
     }
 }
 
+// ===== impl Config =====
+
+impl Config {
+    /// Set the initial window size of the remote peer.
+    pub fn initial_window_size(&mut self, size: u32) -> &mut Self {
+        self.settings.set_initial_window_size(Some(size));
+        self
+    }
+
+    pub fn handshake<T>(&self, io: T) -> Handshake<T, Bytes>
```

Comment:
I wonder if this fn should be removed and `handshake2` renamed to `handshake`. Then `Client::handshake` could be the "simple" option.